packages/docs/site/i18n/gu/docusaurus-theme-classic/navbar.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Some of those texts are used on the Docusaurus UI. You can translate the `message` property to Gujarati

<img width="1561" height="1334" alt="translation" src="https://github.com/user-attachments/assets/8aeee377-30e2-4889-9f06-6b8d9d9e99e7" />